examples/docker-agent/run.py

```diff
@@ -1,22 +1,19 @@
-"""Generic runner for docker agents"""
+"""Implementation of a simple deterministic agent using Docker."""
 
-import os
-import importlib
+import pommerman
+from pommerman.runner import DockerAgentRunner
 
-from common import DockerAgent
 
+class MyAgent(DockerAgentRunner):
+    def __init__(self):
+        self._agent = pommerman.make_agent('ffa_v0', 'test::agents.SimpleAgent')
```

Comment:
make_agent expects an agent_id and will default to -1 if not given. how will this work with that?

Reply:
This has been a slight cause of concern. The `agent_id` is kind of coupled with the creation. Since, it does not interfere with the operations of the agents, shall we do it in a way, that once all agents have been created, then we assign the IDs. Essentially, looping over the `env._agents` list